src/js/events/ChaosEvents.js

```diff
@@ -0,0 +1,6 @@
+var ChaosEvents = {
+  FINISHED: "CHAOS_EVENTS_FINISHED",
+  STARTED: "CHAOS_EVENTS_STARTED"
```

Comment:
No, it's only that fast with your small amount of apps I think. The tasksId are sent to the endpoint only if all taskIds are collected from the apps..